src/qt/reissueassetdialog.cpp

```diff
@@ -638,19 +649,19 @@ bool ReissueAssetDialog::checkIPFSHash(QString hash)
 {
     if (!hash.isEmpty()) {
         std::string error;
-        if (!CheckEncodedIPFS(hash.toStdString(), error)) {
+        if (!CheckEncoded(DecodeAssetData(hash.toStdString()), error)) {
             ui->ipfsText->setStyleSheet(STYLE_INVALID);
-            showMessage(tr("IPFS Hash must start with 'Qm'"));
+            showMessage(tr("IPFS Hash must start with 'Qm', Txid is indicated by 0x54 0x20"));
             disableReissueButton();
             return false;
-        } else if (hash.size() != 46) {
+        } else if (hash.size() != 46 && hash.size() != 64) {
             ui->ipfsText->setStyleSheet(STYLE_INVALID);
-            showMessage(tr("IPFS Hash must have size of 46 characters"));
+            showMessage(tr("IPFS/Txid Hash must have size of 46 characters"));
             disableReissueButton();
             return false;
-        } else if (DecodeIPFS(ui->ipfsText->text().toStdString()).empty()) {
+        } else if (DecodeAssetData(ui->ipfsText->text().toStdString()).empty()) {
             ui->ipfsText->setStyleSheet(STYLE_INVALID);
-            showMessage(tr("IPFS hash is not valid. Please use a valid IPFS hash"));
+            showMessage(tr("IPFS/Txid hash is not valid. Please use a valid IPFS hash"));
```

Comment:
`IPFS/Txid hash is not valid. Please use a valid IPFS or Txid hash`

Reply:
Fixed